datascience-notebook/Dockerfile.onbuild

```diff
@@ -0,0 +1,7 @@
+FROM  jupyter/datascience-notebook
+
+ONBUILD COPY . /home/jovyan/work
+
+ONBUILD RUN  [ -e requirements.txt ] && pip install --no-cache-dir -r requirements.txt
+
+ONBUILD RUN  [ -e environment.yml ] && conda install --file environment.yml
```

Comment:
Should there be a development install line somewhere (e.g. `pip install -e /home/jovyan/work`)?